controllers/memberoperatorconfig/cache.go

```diff
@@ -45,27 +49,50 @@ func loadLatest(cl client.Client, namespace string) error {
 		}
 		return err
 	}
-	configCache.set(config)
+
+	allSecrets, err := loadSecrets(cl)
+	if err != nil {
+		return err
+	}
+
+	configCache.set(config, allSecrets)
 	return nil
 }
 
+func loadSecrets(cl client.Client) (map[string]map[string]string, error) {
+	var allSecrets = make(map[string]map[string]string)
+	secretList := &corev1.SecretList{}
+	err := cl.List(context.TODO(), secretList)
+	if err != nil {
+		return allSecrets, err
+	}
+	for _, secret := range secretList.Items {
```

Comment:
There is a bunch of built-in secrets exist in every namespace. They start with `default-`, `builder-`, `deployer-`.
And they all have an annotation: `kubernetes.io/service-account.name` (with different values though).
We should probably ignore these secrets.

Reply:
I went with the annotation method. Let me know if that should be good: b4189bcf653267d8a8678cd370c8e0764414c7fd